cassiopeia/simulator/Cas9LineageTracingDataSimulator.py

```diff
@@ -73,15 +73,17 @@ class Cas9LineageTracingDataSimulator(LineageTracingDataSimulator):
         size_of_cassette: Number of editable target sites per cassette
         mutation_rate: Exponential parameter for the Cas9 cutting rate. The user
             can either pass in a float, in which every site will mutate at
-            the same rate, or a list with an entry for each cut site indicating
-            potentially a different mutation rate for every site.
+            the same rate, or a list a list of floats of length 
+            `size_of_cassette` or `number_of_cassettes * size_of_cassette`.
```

Comment:
Please specify the expected functionality of each variant of these arguments -- i.e., we assume that if the `float` is passed in then all sites have the same mutation rate; if just a list of `size_of_cassette` then rates differ across a cassette but each cassette is the same etc.